cli/cli_app.go

```diff
@@ -61,6 +62,14 @@ func CreateTerragruntCli(version string) *cli.App {
    Moreover, for the apply and destroy commands, Terragrunt will first try to acquire a lock using DynamoDB. For
    documentation, see https://github.com/gruntwork-io/terragrunt/.`
 
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "config",
+			Value: os.Getenv("TERRAGRUNT_CONFIG"),
```

Comment:
The default value should fall back to `".terragrunt"` if that env var is not set.